geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/AccumuloGeoMesaStats.scala

```diff
@@ -63,27 +61,6 @@ class AccumuloGeoMesaStats(val ds: AccumuloDataStore, statsTable: String, val ge
 
   compactor.run() // schedule initial compaction
 
-  override def getCount(sft: SimpleFeatureType, filter: Filter, exact: Boolean): Option[Long] = {
```

Comment:
Out of curiosity, where did this go?

Reply:
The implementation was in the super class already, but this had some special handling for duplicate rows. Since I remove support for the non-point Z2/Z3 indices, we don't really need the duplicate handling anymore, since none of our indices have duplicates.